examples/purchase-tester/src/main/java/com/revenuecat/purchasetester/OfferingFragment.kt

```diff
@@ -141,39 +141,41 @@ class OfferingFragment : Fragment(), PackageCardAdapter.PackageCardAdapterListen
     }
 
     private fun startPurchaseProduct(
-        purchaseOption: PurchaseOption?,
+        subscriptionOption: SubscriptionOption?,
         currentProduct: StoreProduct,
         upgradeInfo: UpgradeInfo?
     ) {
         when {
-            upgradeInfo == null && purchaseOption == null -> Purchases.sharedInstance.purchaseProductWith(
+            upgradeInfo == null && subscriptionOption == null -> Purchases.sharedInstance.purchaseProductWith(
                 requireActivity(),
                 currentProduct,
                 purchaseErrorCallback,
                 successfulPurchaseCallback
             )
-            upgradeInfo == null && purchaseOption != null -> Purchases.sharedInstance.purchaseSubscriptionOptionWith(
+            upgradeInfo == null && subscriptionOption != null ->
+                Purchases.sharedInstance.purchaseSubscriptionOptionWith(
```

Comment:
NABD but if we are moving this to a new line, should we do the same with the other cases of this `when`?